server/src/services/rsvp.ts

```diff
@@ -25,3 +36,18 @@ export function getRsvpByEventId(eventId: string) {
 export function deleteRsvpById(id: string) {
   return db.prepare('DELETE FROM rsvp WHERE id = ?').run(id);
 }
+
+export function updateRsvpByToken({ eventId, rsvpId, token, name, status, guests }: {
+  eventId: string,
+  rsvpId: string,
+  token: string,
+  name: string,
+  status: string,
+  guests: number
+}) {
+  const sql = `UPDATE rsvp SET name = ?, status = ?, guests = ?, updated_at = ? WHERE id = ? AND event_id = ? AND token = ?`;
+  const now = new Date().toISOString();
```

Comment:
We can get rid of this manual update of `updated_at` by using triggers. However, it seems that SQLite is a bit less featureful with these, with having to use either `WHEN` clauses comparing timestamp values or specifying the columns explicitly, as otherwise it can trigger the trigger recursively 🙃

So it might be easier to keep this for now, sadly...

Reply:
Although maybe the internet is full of liars... 🧐 The `BEFORE UPDATE` trigger seems to have worked fine....

<img width="1276" height="744" alt="Image" src="https://github.com/user-attachments/assets/7b040e96-d49c-4a62-8bc3-ee06dc81ec60" />